src/polymer-language-server.ts

```diff
@@ -87,146 +84,162 @@ connection.onHover(async(textPosition) => {
   return handleErrors(getDocsForHover(textPosition), undefined);
 });
 
-async function getDocsForHover(textPosition: TextDocumentPositionParams): Promise<Hover|undefined> {
-  const localPath = getWorkspacePathToFile(textPosition.textDocument);
-  if (localPath && editorService) {
-    const documentation = await editorService.getDocumentationAtPosition(
-        localPath, convertPosition(textPosition.position));
-    if (documentation) {
-      return {contents: documentation};
+async function getDocsForHover(textPosition: TextDocumentPositionParams):

```

Comment:
Huh. Double check your clang-format version? The desired version is:

```
$ clang-format --version
clang-format NPM version 1.0.45 at /Users/rictic/open/editor-service/node_modules/clang-format/index.js
```

Should be installable via `npm install --global clang-format`